client/app/components/AccountConfigForm.jsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx h */
+import { h } from 'preact'
+
+import Field, { DropdownField } from './Field'
+import AccountLoginForm from './AccountLoginForm'
+
+const folderLink = (folders, currentPath) => {
+  let folder = folders.find(f => f.path + '/' + f.name === currentPath)
+  if (!folder) return
+  return `/apps/files/#folders/${folder.id}`
+}
+
+const AccountConfigForm = ({ t, customView, connectUrl, fields, dirty, error, submit, submitting }) => (
+  <div class={'account-form' + (error ? ' error' : '')}>
+    {fields.folderPath &&
+      <div>
+        <h3>{t('my_accounts location')}</h3>
+        <p>{t('my_accounts location desc')}</p>
+        <DropdownField label={false} {...fields.folderPath} />
+        <a href={folderLink(fields.folderPath.folders, fields.folderPath.value)}>
+          {t('my_accounts location button')}
+        </a>
+      </div>
+    }
+    {fields.calendar &&
+      <div>
+        <h3>{t('my_accounts calendar')}</h3>
+        <p>{t('my_accounts calendar desc')}</p>
+        <Field label={false} {...fields.calendar} />
+      </div>
+    }
+    <h3>{t('my_accounts frequency')}</h3>
+    <p>{t('my_accounts frequency desc')}</p>
+    <Field label={false} {...fields.frequency}>
```

Comment:
Did it make sense to use the dropdown field component here, instead of the simple field? I think it's more appropriate to the frequency fields, what do you think?